<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">QTRunner</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButtonRunTestSuite">
        <property name="text">
         <string>Run test suite!</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="labelTests">
          <property name="text">
           <string>Tests executed:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="labelTestExecutableCounter">
          <property name="text">
           <string>2 of 10</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelOk">
          <property name="text">
           <string>Tests without error:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="labelOKCount">
          <property name="text">
           <string>2</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="labelNOK">
          <property name="text">
           <string>Tests with Error:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="labelNOKCount">
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>0</number>
        </property>
        <property name="textVisible">
         <bool>false</bool>
        </property>
        <property name="format">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>15</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QTableWidget" name="tableWidget">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="showGrid">
          <bool>false</bool>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
        </widget>
        <widget class="QTextEdit" name="textEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="pushButtonAddTest">
          <property name="minimumSize">
           <size>
            <width>128</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>28</height>
           </size>
          </property>
          <property name="text">
           <string>Add test...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonRemoveTest">
          <property name="minimumSize">
           <size>
            <width>128</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>28</height>
           </size>
          </property>
          <property name="text">
           <string>Remove test</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonConfigureTest">
          <property name="minimumSize">
           <size>
            <width>128</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>28</height>
           </size>
          </property>
          <property name="text">
           <string>Configure...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonResetResults">
          <property name="minimumSize">
           <size>
            <width>128</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>28</height>
           </size>
          </property>
          <property name="text">
           <string>Reset results</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>402</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFiles">
    <property name="title">
     <string>Files</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen_project"/>
    <addaction name="separator"/>
    <addaction name="actionSave_project"/>
    <addaction name="actionSave_project_as"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_QTRunner"/>
    <addaction name="separator"/>
    <addaction name="actionCheck_for_updates"/>
   </widget>
   <addaction name="menuFiles"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="QTRunner.qrc">
     <normaloff>:/images/images/filenew.png</normaloff>:/images/images/filenew.png</iconset>
   </property>
   <property name="text">
    <string>New project</string>
   </property>
  </action>
  <action name="actionOpen_project">
   <property name="icon">
    <iconset resource="QTRunner.qrc">
     <normaloff>:/images/images/fileopen.png</normaloff>:/images/images/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>Open project...</string>
   </property>
  </action>
  <action name="actionSave_project">
   <property name="icon">
    <iconset resource="QTRunner.qrc">
     <normaloff>:/images/images/filesave.png</normaloff>:/images/images/filesave.png</iconset>
   </property>
   <property name="text">
    <string>Save project</string>
   </property>
  </action>
  <action name="actionSave_project_as">
   <property name="icon">
    <iconset resource="QTRunner.qrc">
     <normaloff>:/images/images/filesaveas.png</normaloff>:/images/images/filesaveas.png</iconset>
   </property>
   <property name="text">
    <string>Save project as...</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="QTRunner.qrc">
     <normaloff>:/images/images/exit.png</normaloff>:/images/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Alt+F4</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionAbout_QTRunner">
   <property name="icon">
    <iconset resource="QTRunner.qrc">
     <normaloff>:/images/images/QTRunner.png</normaloff>:/images/images/QTRunner.png</iconset>
   </property>
   <property name="text">
    <string>About QTRunner...</string>
   </property>
  </action>
  <action name="actionCheck_for_updates">
   <property name="icon">
    <iconset resource="QTRunner.qrc">
     <normaloff>:/images/images/globe.png</normaloff>:/images/images/globe.png</iconset>
   </property>
   <property name="text">
    <string>Check for updates...</string>
   </property>
  </action>
  <action name="actionUsed_Qt_version">
   <property name="icon">
    <iconset>
     <normaloff>:/images/images/qt.png</normaloff>:/images/images/qt.png</iconset>
   </property>
   <property name="text">
    <string>Used Qt version...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QTRunner.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>263</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
